torchx/c_src/torchx.cpp

```diff
@@ -750,7 +822,7 @@ NIF(cholesky)
 
 NIF(pad)
 {
-  TENSOR_PARAM(0, tensor)
+  TENSOR_PARAM(0, tensor);
   LIST_PARAM(1, std::vector<int64_t>, config)
   SCALAR_PARAM(2, constant)
```

Comment:
```suggestion
  LIST_PARAM(1, std::vector<int64_t>, config);
  SCALAR_PARAM(2, constant);
```

Reply:
i will fix this in a separate PR. and use do-while(0) to wrapper all macro to make sure it in the future.

Reply:
also do you think we should maintain a shared copy of nx_nif_util.h for both torch and exla?